### PR TITLE
get rid of more  null mccs list from countries.json

### PR DESCRIPTION
**Pull request by szjozsef**
In addition to #1421 , to get rid of null entries for mccs, where possible.
Please review and if you consider, then merge this pull request.
Unfortunately It contains some hard-coded values, for country codes which otherwise can't really be obtained by data parsing.
For Cocos Islands and Christmas Island I edited also the Wikipedia page, but if that edit will get reverted for some reason, the included commit will parse the 
"includes
    Cocos Islands (Australia) – CC
    Christmas Island (Australia) – CX
"
list section, including the iso codes from there if they was not already included.
The second commit includes hard-coded iso codes.

### CODE REVIEW

**Review by jwhitlock — CHANGES_REQUESTED** [review]
Thanks @szjozsef, these look like good changes. There's a few changes needed before merging.

File: mobile_codes/tests.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def test_countries_match_mnc_operators(self):
         operator_mccs -= {"901", "902", "991"}
 
         # Exclude FonePlus in British Indian Ocean Territory
-        operator_mccs.remove("995")
+        # operator_mccs.remove("995")
```
[the remark this answers is not in the document]
Comment (szjozsef) [reply mobile_codes/tests.py]: Done

File: mobile_codes/tests.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def test_mcc_multiple_countries_one_code(self):
 
     def test_mcc_multiple_countries(self):
         countries = mobile_codes.mcc("505")
-        assert len(countries) == 2
+        assert len(countries) == 4
```
[the remark this answers is not in the document]
Comment (szjozsef) [reply mobile_codes/tests.py:28]: Done
(else: NF Norfolk Island)

**Review by jwhitlock — APPROVED** [review]
Thanks @szjozsef!

**Pull request merged**